lib/src/sv_internal.h

```diff
@@ -304,6 +304,8 @@ struct _signed_video_t {
   // For signing plugin
   void *plugin_handle;
   sign_or_verify_data_t *sign_data;  // Pointer to all necessary information to sign in a plugin.
+  const char *private_key;
```

Comment:
Add comment what this is used for. A reader might assume that this is the signing key used during the session, but it is not.